content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md

```diff
@@ -9,6 +9,72 @@ menu:
     parent: About the project
 ---
 
+## 1.9.6 [2021-12-xx]
+
+### Features
+
+#### Backup enhancements
+
+- **Revert damaged meta nodes to a previous state**: Add the `meta-only-overwrite-force` option to [`influxd-ctl restore`](/enterprise_influxdb/v1.9/tools/influxd-ctl/#restore) to revert damaged meta nodes in an existing cluster to a previous state when restoring an InfluxDB Enterprise database.
+
+- **Estimate the size of a backup** (full or incremental) and provide progress messages. Add `-estimate` option to [`influxd-ctl backup`](/enterprise_influxdb/v1.9/tools/influxd-ctl/#backup) to estimate the size of a backup (full or incremental) and provide progress messages. Prints the number of files to back up, the percentage of bytes transferred for each file (organized by shard), and the estimated time remaining to complete the backup.
+
+#### Logging enhancements
+
+- **Log active queries when a process is terminated**: Add the [`termination-query-log = false`](/enterprise_influxdb/v1.9/administration/configure/config-data-nodes/#termination-query-log--false) configuration option. When set to `true` all running queries are printed to the log when a data node process receives a `SIGTERM` (for example, a k8s process exceeds the container memory limit or the process is terminated).
+
+- **Log all HTTP requests to meta nodes**. View updated logs when [`cluster-tracing`](/enterprise_influxdb/v1.9/administration/configure/config-meta-nodes/#cluster-tracing--false) is enabled.
+
+### Maintenance updates
+
+#### Build maintenance
+
+- Update to [Flux v0.133](/flux/v0.x/release-notes/#v01330-2021-10-04).
+- Upgrade to Go 1.17.
+- Upgrade miscellaneous dependencies, including `grpc`.
+- Upgrade `protobuf` library.
+- Export package signatures currently being created in Jenkins to AWS s3.
+- Add `go vet` and `goimports` static checks prior to building binaries.
+- Verify rustup script in CI.
+- Fix rust-install script in CI.
+- Update ear-scaffold build instructions on Ubuntu.
```

Comment:
do we want to include this level of detail re "build maintenance"?

Reply:
No, I don't think so.

Reply:
It may make sense to document the protobuf library upgrade, that's something people might want to know about. Otherwise, I agree we don't need build system / deployment changes documented like that.